flask_ad_auth/ad_login.py

```diff
@@ -140,11 +140,15 @@ def is_expired(self):
         return True
 
     def is_in_group(self, group):
-        for g in self.groups:
-            if g["id"] == group:
-                return True
-            if g["name"].lower() == group.lower():
-                return True
+        if isinstance(group, list):
+            for id in group:
```

Comment:
you could do something like

return any(item in List1 for item in List2)

Reply:
ok i will change the code